lib/config/config-ops.js

```diff
@@ -12,7 +12,7 @@
 //------------------------------------------------------------------------------
 
 var debug = require("debug"),
-    environments = require("../../conf/environments"),
+    Environments = require("./environments"),
```

Comment:
Same as above. I though we only capitalize objects that need to be newed. We even have a rule for this.